mobile/app/(app)/(drawer)/(tabs)/(observation)/form-details/[formId].tsx

```diff
@@ -138,14 +138,23 @@ const FormDetails = () => {
           answer = { ...answer, selectionValues: selectedAnswersTexts };
         }
 
+        // sort the notes by createdAt date in order to extract the text of the last added note
+        const sortedNotes =
+          notes && notes[q.id]
+            ? notes[q.id].slice().sort((a, b) => {
```

Comment:
What is the `slice` doing here?

Reply:
it's creating a copy of the notes array, so that the sorting does not happen in place